### PR TITLE
Any API - Correct multiword examples

### PR DESCRIPTION
**Pull request by aelmanaa**
Fix https://docs.chain.link/docs/multi-variable-responses/

### CODE REVIEW

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -47,7 +47,7 @@
```
[the remark this answers is not in the document]
Comment (aelmanaa) [reply yarn.lock:47]: Once I rebased, a new _yarn.lock_ was generated once I ran _yarn install_ .  No changes to _package.json_

File: docs/Using Any API/multi-variable-responses.md [hunk 2]
```diff
@@ -2,59 +2,61 @@
...
     <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
 </div>
 
-The job spec for the Chainlink node in this example can be [found here](../example-job-spec-multi-word/).
-
-If the LINK address for targeted blockchain is not [publicly available](../link-token-contracts/) yet, replace [setPublicChainlinkToken(/)](../chainlink-framework/#setpublicchainlinktoken) with [setChainlinkToken(_address)](../chainlink-framework/#setchainlinktoken) in the constructor, where `_address` is a corresponding LINK token contract.
+Note that this example contract is hardcoded to work on _Kovan_ testnet. The job spec for the Chainlink node in this example can be found [here](/docs/example-job-spec-multi-word/).
 
-## Choosing an Oracle and JobId
+## Choosing Link token address, Oracle and JobId
```
Comment (dwightjl) [inline docs/Using Any API/multi-variable-responses.md:55]:
```suggestion
## Setting the LINK token address, Oracle, and JobId
```

File: docs/Using Any API/multi-variable-responses.md [hunk 3]
```diff
@@ -2,59 +2,61 @@
...
->❗️ Note on Funding Contracts
+> ❗️ Note on Funding Contracts
 >
 > Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
->
+
+Assume that a user wants to obtain the ETH price quoted against three different currencies: _BTC_ , _USD_ and _EUR_. If they use only a single-word job, it would require three different requests. To make that more efficient, they can use multi-word responses to do it all in a single request as shown in the following example:
 
 ```solidity Kovan
 {% include 'samples/APIRequests/MultiWordConsumer.sol' %}
```
Comment (dwightjl) [inline docs/Using Any API/multi-variable-responses.md:45]: Add some steps to make this example work.

File: yarn.lock (hunk 1 above)
[the remark this answers is not in the document]
Comment (dwightjl) [reply yarn.lock:47]: Rebase to main and try it again? I'm not getting any changes on main right now. It seems like the registry URL is changing as well.

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**